server/led_sign/led_sign_server.py

```diff
@@ -38,7 +39,8 @@ def WriteCommandToSign(self, request):
             self.proc.kill()
 
         self.proc = subprocess.Popen(command)
-        
+        LedSignMock(self.sign_data)
```

Comment:
do we want to always show the mock or conditionally show it? e.g.

```
# show it
python server/led_sign/led_sign_server.py --mock-output
# dont show it (default)
python server/led_sign/led_sign_server.py
```

it's up to you, this is a design choice